build.gradle

```diff
@@ -168,18 +168,23 @@ dependencies {
     implementation 'jakarta.annotation:jakarta.annotation-api:2.1.1'
     implementation 'jakarta.inject:jakarta.inject-api:2.0.1'
 
-
-    implementation 'org.jabref:afterburner.fx:1.1.0-SNAPSHOT'
+    implementation('org.jabref:afterburner.fx:1.1.0-SNAPSHOT') {
+        exclude group: 'org.openjfx'
+    }
     implementation 'org.kordamp.ikonli:ikonli-javafx:12.3.1'
     implementation 'org.kordamp.ikonli:ikonli-materialdesign2-pack:12.3.1'
     implementation 'com.github.sialcasa.mvvmFX:mvvmfx-validation:f195849ca9' //jitpack
     implementation 'de.saxsys:mvvmfx:1.8.0'
-    implementation 'com.tobiasdiez:easybind:2.2.1-SNAPSHOT'
+    implementation('com.tobiasdiez:easybind:2.2.1-SNAPSHOT') {
+        exclude group: 'org.javafx'
```

Comment:
same as above, javafx is used transitive https://github.com/tobiasdiez/EasyBind/blob/c48a457fb8ffebc8f42c3d627db65b7494d0b3d2/src/main/java/module-info.java#L6